jina/parsers/orchestrate/pod.py

```diff
@@ -114,6 +114,12 @@ def mixin_pod_parser(parser, pod_type: str = 'worker'):
             default=False,
             help='If set, the Gateway will restart while serving if the YAML configuration source is changed.'
         )
+    gp.add_argument(
+        '--install-requirements',
+        action='store_true',
+        default=False,
+        help='If set, try to install `requirements.txt` from the local Executor if exists in the Executor folder. If using Hub, install `requirements.txt` in the Hub Executor bundle to local.'
+    )
```

Comment:
should not be added to the gateway in my opinion

Reply:
I don't think this is much different from mixin_hub_pull_options_parser which comes from hubble.
I suggest just editing the hubble sdk rather than re-adding the argument